llvm/lib/Target/RISCV/MCTargetDesc/RISCVMachObjectWriter.cpp

```diff
@@ -0,0 +1,56 @@
+//===-- RISCVMachObjectWriter.cpp - RISC-V Mach Object Writer -------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "MCTargetDesc/RISCVFixupKinds.h"
+#include "MCTargetDesc/RISCVMCExpr.h"
```

Comment:
RISCVMCExpr.h has been deleted. This file doesn't need the header.

Reply:
Done